### ISSUE TITLE
Allow to skip a subset of prompts

### ISSUE THREAD

**Issue by eifinger** [astral-sh/rye#620]
We are using https://github.com/eifinger/setup-rye to automatically set up rye in our CICD pipelines.  In this action I am using `RYE_INSTALL_OPTION` to skip all prompts. Since version `0.21.0` this means that `.profile` also gets manipulated. Since rye is not installed into the users home directory but rather to the GitHub Actions toolscache it could mean that the path that gets added to `.profile` might be cleaned up.

On a non-epheremal host like a self hosted runner on bare metal this messes up some things.

It would be great if I can skip/accept a subset of prompts. Either with dedicated environment variables or by setting a set of values into `RYE_INSTALL_OPTION` e.g. `--yes --add-to-path`

**Comment by mitsuhiko** [comment]
Yes, I think this needs a bunch of flags. I think it would reasonable to model this out what `--yes` should apply to and what flags should be opt-in vs opt-out etc.

**Comment by dsp** [comment]
Is it sensible to add a `rye self bootstrap` command with various flags?

**Comment by dsp** [comment]
forget that, there is already `rye self install`
